ui/src/components/ResourceDialog/ProcessorsDialog/InlineProcessorLabel.tsx

```diff
@@ -165,7 +165,10 @@ export const InlineProcessorLabel: React.FC<Props> = ({
       >
         <Stack direction={"row"} spacing={1}>
           <MenuIcon className={styles["hover-icon"]} />
-          <Typography>{data?.processorType?.metadata.displayName}</Typography>
+          <Typography fontWeight={900}>{data?.processorType?.metadata.displayName}</Typography>
```

Comment:
I dont think we actually have fontWeight 900, just 300,400,600, for nunito sans.  We import these in the `index.html` file for reference.
```suggestion
          <Typography fontWeight={600}>{data?.processorType?.metadata.displayName}</Typography>
```